src/configurationProvider.ts

```diff
@@ -382,7 +382,7 @@ export class JavaDebugConfigurationProvider implements vscode.DebugConfiguration
             if (pickItems[positionForActiveEditor].detail) {
                 pickItems[positionForActiveEditor].detail += `, active editor (${path.basename(options[positionForActiveEditor].filePath)})`;
             } else {
-                pickItems[positionForActiveEditor].detail = `$(clock) active editor (${path.basename(options[positionForActiveEditor].filePath)})`;
+                pickItems[positionForActiveEditor].detail = `$(file-text) active editor (${path.basename(options[positionForActiveEditor].filePath)})`;
```

Comment:
I don't see any "file-text" in octicons. Is it [this one](https://octicons.github.com/icon/file/)?

Reply:
vscode doesn't recognize `$(file)`.  Refer to vscode source code about the supported list.
https://github.com/Microsoft/vscode/blob/master/src/vs/base/browser/ui/octiconLabel/octicons/octicons.less#L94